### PR TITLE
add more spacing to room info in room modal

### PR DESCRIPTION
**Pull request by denisdimitrov**
Adds spacing to the room modal information inside.

### CODE REVIEW

File: src/components/templates/PartyMap/components/RoomModal/RoomModal.scss [hunk 1]
```diff
@@ -14,6 +14,14 @@ $spacing: 20px;
     background-size: cover;
   }
 
+  &__room-info {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: $spacing;
```
Comment (goran-peoski-work) [inline src/components/templates/PartyMap/components/RoomModal/RoomModal.scss:21]: Top padding just adds unnecessary space and makes text uneven with image.

```suggestion
    padding: 0 $spacing $spacing;
```

**Review by goran-peoski-work — APPROVED** [review]
Approved with comments:

Could use trimming the top padding of `&__room-info` as well as maybe get the `&__main` bottom padding to lower value or `&__description` top margin while we're at it

sparkle-room-modal-01

**Comment by goran-peoski-work** [comment]
@denisdimitrov Based on https://github.com/sparkletown/internal-sparkle-issues/issues/1167#issue-988025518 and similar comments from @whoaitsaimz, I guess the  button text can be shortened to just _Join_ instead of _Join Room_

**Comment by sunny-viktoryia** [comment]
@denisdimitrov looks like the PR is approved. Are we still waiting for something/someone to get it merged?

**Comment by denisdimitrov** [comment]
@sunny-viktoryia There was another fix merged instead of my PR for some strange reason. I guess this can be closed.

**Comment by sunny-viktoryia** [comment]
ah, I see.. Can you close this one then, please? 🙃

**Comment by denisdimitrov** [comment]
https://github.com/sparkletown/sparkle/pull/2241#issuecomment-916197798

**Pull request closed without merging**